.gitlab/package_build/suse_rpm.yml

```diff
@@ -16,6 +16,7 @@
     - set -x
     - mkdir -p /tmp/system-probe
     - tar -xf $CI_PROJECT_DIR/sysprobe-build-outputs.tar.xz -C /tmp/system-probe
+    - cp -R /tmp/system-probe/* .
```

Comment:
I think the point of using `/tmp/system-probe` and `SYSTEM_PROBE_BIN`/`--system-probe-bin` was that building system-probe was optional? Is that no longer true?

Reply:
The --system-probe-bin parameter seems to be passed to omnibus as SYSTEM_PROBE_BIN to copy the .c and .o files, which is still required. This PR only rebuilds the system-probe binary during the omnibus stage.

Reply:
I'm wondering why we need to copy everything from /tmp/system-probe to `.`, when we extract to `/tmp/system-probe` on the previous line.

Reply:
To get the .go generated files the are required by the system-probe compilation

Reply:
Should we no longer extract to `/tmp/system-probe` and use the source tree instead?

Reply:
We still need the clang compiler and the minified btfs symbols. I updated the PR to uncompress `sysprobe-build-outputs.tar.xz` into the source tree while keeping llvm + the btfs symbols in /tmp/system-probe